### DISCUSSION TITLE
Is 1.19 ready ?

### DISCUSSION THREAD

**Discussion by bananasmoothii** [Minestom/Minestom#1302] [Help]
Hello, apparently the 1.19 branch is ready for use. But could someone provide a little more information?
For example, what about chat signing ? Right now I am using `PlayerChatEvent.setChatFormat`, is there anything to change ?
Are there other points needing attention? A little message or a piece of documentation about that wouldn't hurt 🙃

**Comment by dev-hydrogen** [comment]
Its not production ready, but you can test it.
For example potions cause disconnects and MojangAuth doesnt work.

**Comment by emortaldev** [comment]
I have a fork of Minestom that "fixes" potions because I didn't want to downgrade, and I also didn't need MojangAuth

I haven't noticed any other issues so far. Both issues should also be fixed pretty soon so maybe just wait a little longer (1.19 might even be merged within a few days if we're lucky)

Edit: MojangAuth has been fixed since 3d61dba
